core/src/main/java/net/neoforged/fml/config/ModConfig.java

```diff
@@ -73,7 +75,7 @@ void setConfigData(final CommentedConfig configData) {
         this.spec.acceptConfig(this.configData);
     }
 
-    void fireEvent(final IConfigEvent configEvent) {
+    public void fireEvent(final IConfigEvent configEvent) {
```

Comment:
We should get rid of `dispatchConfigEvent` and the `configHandler`, and instead just fire via `acceptEvent` or the event bus directly.

Reply:
Yes

Reply:
@dima-dencep can you remove `dispatchConfigEvent` and `configHandler` from `ModContainer` as well?

Reply:
Done

Reply:
Everything looks good now, however the fact that the events are in a different subproject stresses me out a bit.... What is the purpose of this?

Reply:
Until the eventbus changes made in 1.20.2, it was necessary for the events to be in the game layer to be processed by the eventbus transformer. This is technically not necessary anymore, but also not a problem in practice? In any case don't move them.